mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -331,3 +331,17 @@ If you were relying on accessing an individual flag by index (for example, `goog
 ### `rule.rate_limit_options.encorce_on_key` no longer has default value
 
 Previously, the default value for `rule.rate_limit_options.encorce_on_key` is "ALL", now this field no longer has a default value.
+
+## Resource: `google_logging_metric`
+
+### Additional fields are now required metrics with `value_type = "DISTRIBUTION"`
```

Comment:
```suggestion
### Additional `bucket_options` subfields are now properly required
```

Reply:
as the PR is written, the `value_type` does not determine if the fields are required.